bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml

```diff
@@ -110,8 +110,6 @@ variables:
   BRANCH_NAME: ${bamboo.planRepository.branchName}
   CMR_OAUTH_PROVIDER: earthdata
   ORCA_API_DEPLOYMENT_INVOKE_URL: REPLACE ME
-  ORCA_RECOVERY_STEP_FUNCTION_ARN: REPLACE ME
-  ORCA_COPY_TO_ARCHIVE_STEP_FUNCTION_ARN: REPLACE ME
   ORCA_RECOVERY_BUCKET_NAME: REPLACE ME
```

Comment:
Should ORCA_RECOVERY_BUCKET_NAME be removed as well?

Reply:
Oops removed.